### PR TITLE
Feature/nearby location from location

### LINKED ISSUE #45: Get nearby items based on Location
Get Location and query geohashes

### PR DESCRIPTION
**Pull request by PassiDel**
Closes #45

(same as #57, but rebased)
- using LocationController
- using distance conversion (1.000m => 1km)

### CODE REVIEW

**Review by henrijaeger — APPROVED** [review]

**Review by frcroth — APPROVED** [review]
Nice that you added tests as well

File: app/src/main/java/app/wefridge/wefridge/NearbyItemFragment.kt [hunk 1]
```diff
@@ -79,62 +94,35 @@ class NearbyItemFragment : Fragment() {
...
-
-                    loading = false
-                }
-                .addOnFailureListener {
-                    Log.e("Auth", "error", it)
-                }
+        if (geoLocation == null)
+            return locationController.getCurrentLocation()
+
+//        Log.v("Auth", "Page: ${lastVisible?.id ?: "new"}")
```
[the remark this answers is not in the document]
Comment (PassiDel) [reply app/src/main/java/app/wefridge/wefridge/NearbyItemFragment.kt:100]: it is needed for the first load and on refresh

[the remark this answers is not in the document]
Comment (frcroth) [reply app/src/main/java/app/wefridge/wefridge/NearbyItemFragment.kt:100]: I was talking about the commented out log statement

[the remark this answers is not in the document]
Comment (PassiDel) [reply app/src/main/java/app/wefridge/wefridge/NearbyItemFragment.kt:100]: ahh, no its not

**Pull request merged**